.gitignore

```
@@ -1,4 +1,6 @@
 Latex/metadaten.sty
+Latex/main.pdf
+Latex/svg-inkscape
 
```

weil du einmal build-related dinge in der gitignore änderst:
```suggestion
.build

```
braucht man für die neue container-version. `latex.out` würde ich als legacy trotzdem erstmal drin lassen und evtl. später entfernen
(frag mich nicht warum das diff so komisch ist)